meshtastic/tests/test_node.py

```
@@ -1247,6 +1311,350 @@ def test_setURL_add_only_raises_when_no_disabled_slot_available(
+    disabled1 = Channel(index=1, role=Channel.Role.DISABLED)
+    disabled2 = Channel(index=2, role=Channel.Role.DISABLED)
+    anode.channels = [primary, disabled1, disabled2]
+    anode.localConfig.lora.hop_limit = 3
+    before_snapshot = [channel.SerializeToString() for channel in anode.channels]
+
+    staged_writes: list[tuple[int, str, int | None]] = []
+    rollback_writes: list[tuple[int, int | None]] = []
+
+    def _send_admin_with_staged_write_failure(
+        msg: admin_pb2.AdminMessage,
+        wantResponse: bool = False,
```

Ruff (0.15.5)

[error] Boolean-typed positional argument in function definition

(FBT001)